src/components/canvas.tsx

```diff
@@ -48,7 +50,8 @@ class Canvas extends React.Component<CanvasProps, CanvasState> {
   render() {
     return (
       <div className="card-builder-right">
-        <canvas ref="canvas" className="card" css={{ display: this.state.showCanvas ? "block" : "none" }}></canvas>
+        <canvas ref="canvas" className="card canvas-layers" css={{ display: this.state.showCanvas ? "block" : "none" }}></canvas>
```

Comment:
I would perhaps wrap the canvases in a div and add the conditional styling to that instead of duplicating it.